packages/blockchain-wallet-v4-frontend/src/scenes/Home/Banners/NewCurrency/index.tsx

```diff
@@ -0,0 +1,126 @@
+import { bindActionCreators, Dispatch } from 'redux'
+import { connect } from 'react-redux'
+import { FormattedMessage } from 'react-intl'
+import React from 'react'
+import styled from 'styled-components'
+
+import { actions, selectors } from 'data'
+import { Button, Icon, Text } from 'blockchain-info-components'
+import { CoinType, WalletCurrencyType } from 'core/types'
+import { Wrapper } from '../styles'
+
+const StyledWrapper = styled(Wrapper)`
+  align-items: center !important;
+  font-family: 'Inter', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto,
```

Comment:
why do we need to define `font-family`?

Reply:
When I created the `StyledWrapper` the font was a serif font. I can dig into exactly why but in order to move forward I just found the main font we used for the app and put it there.